slasher/detection/service.go

```diff
@@ -196,6 +196,10 @@ func (ds *Service) detectHistoricalChainData(ctx context.Context) {
 				}
 			}
 			ds.submitAttesterSlashings(ctx, slashings)
+
+			if err := ds.UpdateHighestAttestation(ctx, att); err != nil {
+				log.WithError(err).Errorf("could not update highest attestation")
+			}
 		}
 		latestStoredHead = &ethpb.ChainHead{HeadEpoch: epoch}
 		if err := ds.slasherDB.SaveChainHead(ctx, latestStoredHead); err != nil {
```

Comment:
Code seems to be untested (no unit tests covering it):
![image](https://user-images.githubusercontent.com/188194/96448274-cd236d00-121b-11eb-88c9-f0b03374e8dc.png)

(red lines denote uncovered code)

Reply:
Not sure any of function in service.go are tested..